Source/WebKit/UIProcess/ios/PageClientImplIOS.mm

```diff
@@ -971,7 +971,6 @@ static UIInterfaceOrientationMask toUIInterfaceOrientationMask(WebCore::ScreenOr
     }
 
     [m_webView _showPasswordViewWithDocumentName:fileName passwordHandler:passwordHandler.get()];
-    NavigationState::fromWebPage(*m_webView.get()->_page).didRequestPasswordForQuickLookDocument();
```

Comment:
Is a `_didRequestPasswordForDocument` call being made elsewhere?

Reply:
It moved into _showPasswordViewWithDocumentName